tests/conftest.py

```diff
@@ -454,3 +455,34 @@ def mocked_setup_pub_logger(mocker):
 def mocked_setup_sub_logger(mocker):
     return mocker.patch(
         'bigchaindb.log.setup.setup_sub_logger', autospec=True, spec_set=True)
+
+
+@pytest.fixture
+def bigchaindb_blackbox():
+    def wait_for_connection(url):
+        while True:
+            try:
```

Comment:
We should probably only try a couple of times so that this does not run forever.